system/templates/rolls/preroll-dialog.hbs

```diff
@@ -0,0 +1,71 @@
+{{#*inline "statDescription"}}
+{{#if (eq Using.length 1)}}
+roll +{{lookup Using 0}}
+{{else}}
+roll {{Method}} of {{join Using delimiter=', '}} {{!-- TODO: i18n --}}
+{{/if}}
+{{/inline}}
+
+<form class="preroll-dialog">
+  {{#if move}}
+  <div>{{{enrichMarkdown move.data.data.Trigger.Text}}}</div>
+
+  {{#if move.data.data.Trigger.Options}}
+  <ul>
+    {{#each move.data.data.Trigger.Options}}
+    {{#if Text}}
+    <li>
+      {{ Text }}: <strong>{{>statDescription}}</strong>
+    </li>
+    {{/if}}
+    {{/each}}
+  </ul>
+  {{/if}}
+  {{/if}}
+
+  <div class="dice-roll ironsworn-roll">
+    {{{graphic}}}
+  </div>
+
+  {{#if action}}
+  <div class="form-group">
+    <label>{{localize 'IRONSWORN.Adds'}}</label>
+    <input type="number" step="1" name="adds" value="0" />
+  </div>
+  {{/if}}
+
+  {{!-- TODO: simple options from the move + actor assets --}}
+
+  <details style="padding: 5px; margin: 5px 0; border: 1px solid">
+    <summary>{{localize 'IRONSWORN.RollDialog.AdvancedRollOptions'}}</summary>
+
+    <div class="form-group">
+      <label class="checkbox">
+        <input type="checkbox" name="automaticOutcome">
+        {{localize 'IRONSWORN.RollDialog.PredeterminedOutcome'}}
+      </label>
+      <select name="automaticOutcomeValue">
```

Comment:
needs a label (`aria-label`, `aria-labelled`, or `<label>` element)

Reply:
Again, let's come back to this for v2. I don't want to bog this PR down with too much rework for semantic correctness; this is following Foundry norms, and it works better than what we have now.